applications/nrf_desktop/configuration/nrf52833dk_nrf52820/prj.conf

```diff
@@ -0,0 +1,180 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application Configuration
+
+CONFIG_DESKTOP_INIT_LOG_MOTION_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_EVENT=n
+CONFIG_DESKTOP_INIT_LOG_HID_REPORT_SENT_EVENT=n
+CONFIG_CAF_INIT_LOG_KEEP_ALIVE_EVENTS=n
+
+CONFIG_DESKTOP_HID_REPORT_DESC="configuration/common/hid_report_desc.c"
+
+CONFIG_CAF=y
+CONFIG_CAF_PM_EVENTS=y
+CONFIG_CAF_POWER_MANAGER=y
+CONFIG_CAF_POWER_MANAGER_STAY_ON=y
+CONFIG_CAF_BUTTONS=y
+CONFIG_CAF_BUTTONS_POLARITY_INVERSED=y
+CONFIG_CAF_BUTTONS_PM_KEEP_ALIVE=n
+
+CONFIG_CAF_CLICK_DETECTOR=y
+
+CONFIG_CAF_LEDS=y
+CONFIG_CAF_LEDS_GPIO=y
```

Comment:
Please keep configurations for various build types consistent. Either remove it here or add to other configurations of this board.